### ISSUE TITLE
No more free-tier for API key

### ISSUE THREAD

**Issue by batpigandme** [ALShum/rwunderground#44]
Just an FYI, since I went to get an API key to use your package this morning:
> To improve our services and enhance our relationship with our users, we will no longer provide free weather API keys as part of our program. If you have been directed to download our Weather Underground free API key by a third party provider, please contact your vendor for resolution.

https://www.wunderground.com/weather/api

**Comment by erichare** [comment]
Mara, thank you! Yeah it's unfortunate - I think in the meantime until there's a better solution, we'll just have to put up a disclaimer in the package about this... in order for it to work, an API key is needed which means right now either paying, or already having access to a previous API key. Not ideal for sure 😞 I'll leave this issue open to discussion in the meantime.

**Comment by batpigandme** [comment]
@erichare Yeah, that was my thought, too. Bummer, especially since this package is PERFECT for a (silly, just-for-fun, not-shelling-out-💰-during) project I'm working on!

**Comment by madsboending** [comment]
Bummer! Do you guys know of any free alternatives for getting historical weather data in R?

**Comment by erichare** [comment]
I'm not familiar with any myself. I'll be researching the same thing. It's an unfortunate development for all of us as we really did depend on this functionality for various projects. Hopefully an alternative will pop up...

**Comment by madsboending** [comment]
Maybe check out openweathermap.org, they offer a free access to their api that might do the trick...
